src/System.Security.Cryptography.Pkcs/src/Resources/Strings.resx

```diff
@@ -100,4 +100,7 @@
   <data name="InvalidOperation_WrongOidInAsnCollection" xml:space="preserve">
     <value>AsnEncodedData element in the collection has wrong Oid value: expected = '{0}', actual = '{1}'.</value>
   </data>
+  <data name="PlatformNotSupported_CryptographyPkcs" xml:space="preserve">
+    <value>System.Security.Cryptography.Pkcs is not yet supported on this platform</value>
```

Comment:
I don't know about the rest of the libraries, but the other crypto ones are "this is permanent".  The Pkcs library is just point in time, we want it supported on all platforms.